docker-compose.yml

```diff
@@ -72,7 +72,13 @@ services:
         ports:
             - 5432:5432
         environment:
-            POSTGRES_USER: xpub
+            POSTGRES_USER: ${PGUSER}
+            POSTGRES_DB: ${PGDATABASE}
```

Comment:
I cannot fine `POSTGRES_DB` used in any of the repos

Reply:
It's used by the postgres container. It will create a database called `$POSTGRES_DB`.

https://hub.docker.com/_/postgres/

The plan is to run the `setup-database.sh` script for both local and remote database instances, so this allows the script to pick up database details from `PG.*` environment variables and use the same details when setting up any local instances.

Reply:
Worth it to add a comment that `POSTGRES_` variables are for the local server, while `PG*` ones are for `psql`